cluster/cluster.yaml

```diff
@@ -876,17 +819,9 @@ Resources:
               - Action:
                   - kms:Decrypt
                 Effect: Allow
-{{- if and (eq .Cluster.ConfigItems.deployment_secret_key_managed "true") (ne .Cluster.Environment "e2e") }}
-                Resource:
-                  - !GetAtt DeploymentSecretKey.Arn
-{{- else }}
-                Resource: "*"
-{{- if and (eq .Cluster.ConfigItems.deployment_secret_decrypt_any "false") (ne .Cluster.Environment "e2e") }}
```

Comment:
`Resource: "*""` is missing. That should still be there along with the StringLike condiation.

Reply:
Fixed.